core/storage/user/gae_models.py

```diff
@@ -643,3 +643,63 @@ class UserBulkEmailsModel(base_models.BaseModel):
     # IDs of all BulkEmailModels that correspond to bulk emails sent to this
     # user.
     sent_email_model_ids = ndb.StringProperty(indexed=True, repeated=True)
+
+
+class UserSkillMasteryModel(base_models.BaseModel):
+    """Model for storing a user's degree of mastery of a skill in Oppia.
+
+    This model stores the degree of mastery of each skill for a given user.
+
+    The id for this model is of form '{{USER_ID}}:{{SKILL_ID}}'
+    """
+
+    # The model id for the user and the skill.
+    id = ndb.StringProperty(required=True)
```

Comment:
You don't need this. All ndb.Model classes already have an id field.

Reply:
Fixed.